persistent-scratch.el

```diff
@@ -154,7 +154,8 @@ representing the time of the last `persistent-scratch-new-backup' call."
           (let ((coding-system-for-write 'utf-8-unix))
             (write-region str nil tmp-file nil 0))
         (set-default-file-modes old-umask)))
-    (rename-file tmp-file actual-file t))
+    (when (interactive-p)
```

Comment:
`interactive-p` is obsolete, should be replaced with `called-interactively-p`.

Reply:
Should be fixed with new patch.